src/dotnet/APIView/APIViewJsonValidator/Program.cs

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.CommandLine.Invocation;
+using System.CommandLine;
+using System.IO.Compression;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Xml.Linq;
+using APIView;
+using ApiView;
+
+public static class Program
+{
+    public static int Main(string[] args)
+    {
+        var jsonFilePath = new Option<FileInfo>("--path", "Path to the input json file").ExistingOnly();
+        jsonFilePath.IsRequired = true;
+        var outputDir = new Option<DirectoryInfo>("--outputDir", "Path to the output Directory(Optional).");
+        var dumpOption = new Option<bool>("--dumpApiText", "Dump the API text to a txt file.");
+        var convertOption = new Option<bool>("--convertToTree", "Convert old APIView token model to new tree token model.");
```

Comment:
I wonder if it will be better to use the same terminology as the documentation https://github.com/Azure/azure-sdk-tools/blob/main/tools/apiview/parsers/CONTRIBUTING.md
```suggestion
        var convertOption = new Option<bool>("--convertToTree", "Convert APIView flat token model into a parent - child token model");
```